Add `instance` property to req/rep prototypes (#3102)
fastify/fastify: docs/Reply.md

  - [Introduction](#introduction)
  - [.code(statusCode)](#codestatuscode)
  - [.statusCode](#statusCode)

>   - [.instance](#instance)

Reviewer: I think `.instance` is going to be confusing. Is it the instance of the server or the instance of the request/reply object? It's ambiguous. What do we think about `.server` instead?

Reviewer 2: I prefer .server as well

- `.code(statusCode)` - Sets the status code.
- `.status(statusCode)` - An alias for `.code(statusCode)`.
- `.statusCode` - Read and set the HTTP status code.

> - `.instance` - A reference to the fastify instance object.

Reviewer 3: ```suggestion
- `.server` - A reference to the fastify instance object.
```

Last remaining.

}
```

<a name="server"></a>
### .server

> Contains a reference to the server object of the current scope.

Reviewer: ```suggestion
The Fastify server instance, scoped to the current [encapsulation context](Encapsulation.md).
```
